config/initializers/default_url_options.rb

```diff
@@ -1,7 +1,7 @@
 default_domains = {
   test: 'localhost:3000',
   development: 'localhost:3000',
-  staging: 'tvs.staging.dxw.net',
+  staging: 'teaching-vacancies-staging.london.cloudapps.digital',
```

Comment:
for now we should access staging still via tvs.staging.dxw.net, please remove this change